{"name": "test_get_customer_by_email", "status": "broken", "statusDetails": {"message": "NameError: name 'generate_random_string' is not defined", "trace": "@pytest.fixture\n    def test_customer_data():\n        return {\n            \"firstname\": \"Test\",\n            \"lastname\": \"User\",\n>           \"email\": generate_random_email(),\n            \"telephone\": generate_random_phone(),\n            \"password\": \"hashed_pass\",  # Предполагем, что пароль уже хэширован\n            \"newsletter\": 0,\n            \"status\": 1,\n        }\n\ntests/test_customer.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def generate_random_email():\n>       return f\"{generate_random_string()}@example.com\"\nE       NameError: name 'generate_random_string' is not defined\n\ntests/test_customer.py:14: NameError"}, "start": 1746287017662, "stop": 1746287017662, "uuid": "f97b6a11-89df-4ccf-b556-60ee15a4b865", "historyId": "957986180b1b18167621813b99c46b38", "testCaseId": "957986180b1b18167621813b99c46b38", "fullName": "tests.test_customer#test_get_customer_by_email", "labels": [{"name": "story", "value": "Поиск клиента по email"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "Клиенты"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_customer"}, {"name": "host", "value": "MacBook-Air-2.local"}, {"name": "thread", "value": "14632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_customer"}]}